docs/installation.md

```diff
@@ -18,7 +18,7 @@ This will be the user responding to commands in the reviews repo.
   ![GitHub's signup page](./images/signup.png "GitHub's signup page")
 
 
-**2.** Go to `Settings >> Developer settings >> Personal access tokens` and create a new token with at least this scopes: `public_repo`, `repo:invite`, `read:org` and `read:user`. Save that token, it will be your `BUFFY_GH_ACCESS_TOKEN`.
+**2.** Go to `Settings >> Developer settings >> Personal access tokens` and create a new token with at these scopes: `public_repo`, `repo:invite`, `read:org`, `read:user` and, if using the approval responder, `admin:org` (as the app will create teams and invite members to the ropensci organization). Save that token, it will be your `BUFFY_GH_ACCESS_TOKEN`.
```

Comment:
```suggestion
**2.** Go to `Settings >> Developer settings >> Personal access tokens` and create a new token with these scopes: `public_repo`, `repo:invite`, `read:org`, `read:user` and, if using the approval responder, `admin:org` (as the app will create teams and invite members to the ropensci organization). Save that token, it will be your `BUFFY_GH_ACCESS_TOKEN`.
```